dimes/dimes.py

```diff
@@ -132,12 +140,31 @@ def finalize_plot(self):
                                     ),
                                     name=time_series.name,
                                     yaxis=f"y{axis_id}",
-                                    mode="lines",
+                                    mode="lines"
+                                    if all(
+                                        variables is None
+                                        for variables in (
+                                            time_series.marker_size,
+                                            time_series.marker_symbol,
+                                            time_series.marker_line_color,
+                                            time_series.marker_fill_color,
+                                        )
+                                    )
+                                    else "lines+markers",
```

Comment:
This can be set up in the new class.

Reply:
@nealkruis I created a new class called LineMode to solve this issue.